aws/b2b_personalization/modules/rds/main.tf

```
@@ -0,0 +1,50 @@
+module "security_group" {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-security-group?ref=v4.16.0"
+
+  name        = "db-security-group-${var.name}"
+  description = "Security group for the Database"
+  vpc_id      = var.vpc_id
+
+  ingress_cidr_blocks = var.subnets
+
+  ingress_with_cidr_blocks = [
+    {
+      rule        = "postgresql-tcp"
+      description = "Postgres port"
+    }
+  ]
+
+  tags = var.tags
+}
+
+resource "aws_db_subnet_group" "private_subnets" {
+  name       = "private-subnets"
+  subnet_ids = toset(var.subnets)
+  tags       = var.tags
+}
+
+resource "aws_db_instance" "postgres" {
+  identifier                      = "${var.name}-postgres-db"
```

we can also just use `var.name` and add the `-postgres-db` part in `_env/rds.hcl`
```suggestion
  identifier                      = var.name
```